docs/old-reference-guide/modules/ROOT/pages/serialization.adoc

```diff
@@ -168,6 +228,17 @@ In that case, it is probably wisest to create a custom serialization mechanism.
 The `DataInputStream` and `DataOutputStream` allow you to easily write the contents of the events to an output stream.
 The `ByteArrayOutputStream` and `ByteArrayInputStream` allow writing to and reading from byte arrays.
 
+=== AvroSerializer
+
+Avro Serializer requires a `SchemaStore` to operate, which must be able to resolve schemas of all messages being processed. By default, you can use default `org.apache.avro.message.SchemaStore.Cache` and add Avro Schema to it manually. If you are using
+SpringBoot, the autoconfiguration will create a `SchemaStore` instance for you. This Schema Store can be pre-filled with Avro Schemas extracted from Avro `SpecificRecordBase` generated by `avro-maven-plugin` located on your classpath. For this purpose,
+a special annotation `@AvroSchemaScan` should be put to any SpringBoot configuration class (similar to SpringData `@EntityScan`) in a package hierarchy including the `SpecificRecordBase` classes.
+
+In case you want to use a central Schema Registry in your application landscape, you might want to provide your own `SchemaStore`
+implementation, accessing this remote registry. Just implement the corresponding interface on your own and wire it into the configuration of the `AxonSerrializer`, but consider to implement a caching strategy, since the `AvroSerializer` will look the Avro Schema by its fingerprint on any serialization / deserialization operation, making fast and local in-memory resolution important to keep up the performance.
```

Comment:
```suggestion
In case you want to use a central Schema Registry in your application landscape, you should provide your own `SchemaStore`
implementation, accessing this remote registry. Just implement the corresponding interface and wire it into the configuration of the `AvroSerrializer`. When doing this, do consider to implement a caching strategy in your implementation. The `AvroSerializer` will check the Avro Schemas by their fingerprints on any serialization / deserialization operation, making fast and local in-memory resolution important to keep up the performance.
```